packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,9 +1,75 @@
-# Change Log
+# Changelog
 
-All notable changes to the "teamsfx-toolkit" extension will be documented in this file.
+## 2.0.0 - May 25 2021
+2.0.0 is a major version upgrade for Teams Toolkit with many new improvements and supports.
+- Add: new design of UI and command palette list
+- Add: new getting started guide, samples and doc link from toolkit
+- Add: new sign-in experience for M365 and Azure
+- Add: new interactive flow for screating new app
+- Add: add message extension capability to Teams app
+- Add: sample codes for in-meeting app.
+- Add: add Typescript support
```

Comment:
Typescript was supported in 1.2

Reply:
it was in our Wiki's release notes, yes i can see previous changelog has it, so I'll remove this from our v2 changelog.